src/components/admin/MainIssueController.jsx

```diff
@@ -179,7 +248,7 @@ export default class MainIssueController extends FalcorController {
           return;
         }
         // The issue is valid, we can publish it
-        this.safeSetState({ publishing: true });
+        this.safeSetState({ publishing: true, published_at: new Date() });
```

Comment:
Can we make this so that it makes sure the date is in the UAE time zone? Might be easier using moment and that probably has a `toDate` method or something like that? May also be you can do it in the context of the native Javascript date

Reply:
it ends up being stored as a numeric UTC timestamp so the date object itself here doesn't really matter - I can check in the db.js file whether the date formatting uses local time or UAE time though

Reply:
I know it uses local time on the server, but we make sure the server time is set to UAE time, it wouldn't hurt explicitly setting it to UAE time in the code though! So feel free to change it server side if you wish